tt_eager/tensor/tensor_utils.hpp

```diff
@@ -75,6 +75,14 @@ void apply(const Tensor& tensor, std::function<void(const Tensor&)> callable);
 // Given a multi-device tensor, return all the devices it is mapped to.
 std::vector<Device*> get_devices(const Tensor& multi_device_tensor);
 
+uint32_t num_buffers_in_tensor(const Tensor& tensor);
```

Comment:
Can you add some documenting comment to say these are helper methods to fetch out the associated attribute info out of multidev-host/device tensors?

Reply:
Added docs for setters/getters.